tests/core/variant/test_array.h

```diff
@@ -129,6 +129,170 @@ TEST_CASE("[Array] has() and count()") {
 	CHECK(arr.count(2) == 0);
 }
 
+TEST_CASE("[Array] find()") {
+	Array arr;
+	// Find on empty array returns nothing
```

Comment:
```suggestion
	// Find on empty array returns nothing.
```
Per the [comment style](https://docs.godotengine.org/en/latest/contributing/development/code_style_guidelines.html#comment-style-guide). For all comments in your addition